backend/app/services/transcription_service.py

```
@@ -0,0 +1,143 @@
+    ".mp3",
+    ".m4a",
+    ".ogg",
+    ".flac",
+    ".aac",
+    ".webm",
+}
+
+WHISPER_MODEL_SIZE = os.getenv("WHISPER_MODEL_SIZE", "tiny")
+WHISPER_DEVICE = os.getenv("WHISPER_DEVICE", "cpu")
+WHISPER_COMPUTE_TYPE = os.getenv("WHISPER_COMPUTE_TYPE", "int8")
+WHISPER_DOWNLOAD_ROOT = os.getenv("WHISPER_DOWNLOAD_ROOT", "/tmp/faster-whisper-cache")
```

Ruff (0.15.5)

[error] Probable insecure usage of temporary file or directory: "/tmp/faster-whisper-cache"

(S108)